src/components/blockHelpers.tsx

```diff
@@ -114,12 +134,13 @@ export const SectionWrapper = styled('section')(
   }) => ({
     ...getSectionSizeStyle(size),
     ...getColorStyles(color),
-    ...(backgroundImage !== 'none' && {
-      backgroundImage: `url(${backgroundImage})`,
-      backgroundSize: 'cover',
-      backgroundPosition: 'center center',
-      backgroundRepeat: 'no-repeat',
-    }),
+    ...(backgroundImage !== 'none' &&
```

Comment:
maybe we could move this logic to its own function, there's a bit of duplicate code here